storage/redis.go

```diff
@@ -131,8 +131,15 @@ func BootRedis(path string, sock string) (func(), error) {
 			}
 		}()
 	}
-
-	secs := 60
+	secs_env, isset := os.LookupEnv("FAKTORY_REDIS_TIMEOUT")
+	secs := 120
```

Comment:
```suggestion
	timeout := 120
```

Reply:
I dont think I need to change variable name that is already initialized in the code before and used later - I just added the feature